Meden-Agan-27/travel-worms
# Front end kick off

What I did:
- created a home page with a footer, a banner 
- adding container, row to devise views and centered them

How to test: 
- localhost: homepage and sign_up sign_in pages.
- thanks to check also the scss and where the files are saved, maybe I made something wrong. I'm not actually sure for the shared folder in views. If you have some refactoring advice, I'm totally open to them!!
